js/scorm/wrapper.js

```diff
@@ -511,12 +540,18 @@ class ScormWrapper {
 
     switch (error.name) {
       case CLIENT_COULD_NOT_CONNECT:
-        notify.popup({
-          _isCancellable: false,
-          title: messages.title,
-          body: message
-        });
-        return;
```

Comment:
Was the return intentially remove?

Reply:
Yes, this was intentional so we can additionally see the logger and debug window, if enabled.